openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -458,3 +458,70 @@ export function initEdit() {
         }, 1000);
     }
 }
+
+const isbn_invalid_checksum = 'Invalid Isbn checksum digit';
```

Comment:
```suggestion
const isbn_invalid_checksum = 'Invalid ISBN checksum digit';
```
Capitalize abbreviation "ISBN".